pkg/scheduler/api/node_info/node_info.go

```diff
@@ -637,8 +637,9 @@ func getNodeGpuMemory(node *v1.Node) (int64, bool) {
 		gpuMemoryLabelValue = convertBytesToMib(gpuMemoryLabelValue)
 	}
 
-	gpuMemoryInMb := convertMibToMb(gpuMemoryLabelValue)
-	return gpuMemoryInMb - (gpuMemoryInMb % 100), true // Floor the memory count to make sure its divided by 100 so there will not be 2 jobs that get same bytes
+	// Use the original MiB value from the node label, don't convert to MB
+	// The flooring should be done in MiB, not MB to preserve accuracy
```

Comment:
:+1:
This comment only makes sense in the PR context, you can remove it from the final version.